iron-form.html

```diff
@@ -272,6 +278,9 @@
 
       /**
        * Submits the form.
+       *
+       * @param {!Event} event
```

Comment:
Looks like this should be `Event=` instead, as it reads like an optional parameter.